packages/code-studio/src/redux/selectors.js

```diff
@@ -0,0 +1,14 @@
+// Session
+/**
+ * Retrieve the current session and it's config
+ * @param {Store} store The redux store that is used
+ * @returns {{ session: dh.IdeSession, config: { type: string }}} The active session and it's options
```

Comment:
I think there's also `id` in the `config`. Maybe use the `SessionConfig` type here?
TBH, `getSession` returning an object that is not a session is a bit confusing. I can't come up with a better name, ok to leave as is unless you have any suggestions.

Reply:
I do agree that `getSession` returning not a session is confusing... `getSessionWrapper`?

Reply:
`getSessionWrapper`